config/prow/config.yaml

```diff
@@ -425,6 +430,7 @@ tide:
     - kubernetes/cluster-bootstrap
     - kubernetes/code-generator
     - kubernetes/component-base
+    - kubernetes/controller-manager
```

Comment:
looks like this is followup to https://github.com/kubernetes/test-infra/pull/17712 as part of https://github.com/kubernetes/org/issues/1895